amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -537,9 +542,43 @@
          </build>
       </profile>
 
+      <profile>
+         <id>build-docker-images</id>
+         <!-- builds "image:latest" locally -->
+         <build>
+            <plugins>
+               <plugin>
+                  <groupId>io.fabric8</groupId>
+                  <artifactId>docker-maven-plugin</artifactId>
+                  <configuration>
+                     <images>
+                        <image>
+                           <name>${image.name}:${image.tag}</name>
+                           <build>
+                              <args>
+                                 <BASE_IMAGE>alfresco/alfresco-community-repo-base:${image.tag}</BASE_IMAGE>
```

Comment:
Is there a reason why `alfresco/alfresco-community-repo-base` has not been replaced by [`${base.image}`](https://github.com/Alfresco/alfresco-community-repo/pull/1656/files#diff-403d6c74e7ee95df44c5858d6b585466b9cc97297191922716e739e6cb9bc519R20) instead?

Reply:
You are right - I changed it to: ${base.image}:${image.tag}